net/crowdsec/patches/000-fix_arm64_makefile.patch

```diff
@@ -0,0 +1,12 @@
+--- a/Makefile
```

Comment:
This patch isn't necessary since the default Go package build doesn't use the upstream makefile.

Reply:
yes, no more needed
removed
thanks